docs/reference/commandline/service_ls.md

```diff
@@ -103,6 +104,34 @@ ID            NAME   MODE        REPLICAS  IMAGE
 0bcjwfh8ychr  redis  replicated  1/1       redis:3.0.6
 ```
 
+## Formatting
+
+The formatting options (`--format`) pretty-prints services output
+using a Go template.
+
+Valid placeholders for the Go template are listed below:
+
+Placeholder | Description
+------------|------------------------------------------------------------------------------------------
+`.ID`       | Service ID
+`.Name`     | Service name
+`.Mode`     | Service mode (replicated, global)
+`.Replicas` | Service replicas
```

Comment:
I think it is ok to introduce all `.Replicas`, `.RunningReplicas`, and `.AllReplicas`.
But it is ok to leave it alone for now.